ui/tests/integration/components/mfa-login-enforcement-form-test.js

```diff
@@ -241,7 +241,12 @@ module('Integration | Component | mfa-login-enforcement-form', function (hooks)
       } else {
         const key = target.label === 'Authentication method' ? 'auth-method' : 'accessor';
         const value = target.label === 'Authentication method' ? 'userpass' : 'auth_userpass_1234';
-        await fillIn(`[data-test-mlef-select="${key}"] select`, value);
+        if (key === 'accessor') {
+          // glimmerized component that does not bind the data-test to the hbs file, and instead has it on the select inside the component
```

Comment:
So this is targeting the `select` element which is nested underneath the element with the test selector applied which should work with `fillIn`. It should work the same as the original implementation since classic components automatically add a wrapping div and apply attributes. I'm fine with leaving as is but happy to help get it working for future reference as well.